### PR TITLE
Backlog 16222 sitemap url settings

### PR DESCRIPTION
**Pull request by gflores-jahia**
## JIRA

https://jira.jahia.org/browse/BACKLOG-16222

## Description

Modify how sitemap URL is handled in the Sitemap UI panel and take URL settings props into account. For this we, check if urlRewriteSeoRulesEnabled is disabled and display render site URL from the backend:

image

- Refactor sitemap index link to a separate component and css
- Add graphql extension to fetch site URL and URL settings
- Add fetch of site URL and URL settings in SitemapIndexLInk component

### CODE REVIEW

File: src/javascript/components/SitemapPanel.scss [hunk 1]
```diff
@@ -33,67 +33,12 @@
     padding-bottom: 30px;
 }
 
-.sitemapIndexFileTitle {
```
Comment (gflores-jahia) [inline src/javascript/components/SitemapPanel.scss:36]: Refactored sitemap index file related css to `panelSections/SitemapIndexLink/SitemapIndexLink.scss`

File: src/javascript/components/Snackbar/Snackbar.jsx [hunk 2]
```diff
@@ -6,7 +6,7 @@ import {Button, Close} from '@jahia/moonstone';
 import styles from './Snackbar.scss';
 
 export const SnackbarComponent = ({
-    open,
+    isOpen,
```
Comment (gflores-jahia) [inline src/javascript/components/Snackbar/Snackbar.jsx:9]: Fixed lint warning on boolean naming

File: src/javascript/components/gqlQueries.js [hunk 3]
```diff
@@ -13,20 +13,6 @@ const GetNodeMixin = gql`
     }
 `;
 
-const GetProperties = gql`
```
Comment (gflores-jahia) [inline src/javascript/components/gqlQueries.js:16]: cleaned up. I didn't see any calls to this one.

File: src/javascript/hooks/graphql/useGetSitemapUrl.js [hunk 4]
```diff
@@ -0,0 +1,33 @@
+import {useEffect, useState} from 'react';
+import {useLazyQuery} from '@apollo/react-hooks';
+import {GetSitemapUrl} from '../../components/gqlQueries';
+
+const useGetSitemapUrl = siteKey => {
```
Comment (AKarmanov) [inline src/javascript/hooks/graphql/useGetSitemapUrl.js:5]: Not a big deal in this case but it looks like for this one we could get away with a standard useQuery({variables : {siteKey}}), it should be smart enough to recognize that query needs to run again, unless I misunderstand something.

**Review by AKarmanov — DISMISSED** [review]

File: src/javascript/hooks/graphql/useGetSitemapUrl.js (hunk 4 above)
Comment (gflores-jahia) [reply src/javascript/hooks/graphql/useGetSitemapUrl.js:5]: Looks like no difference between lazyQuery and standard useQuery. Switched to useQuery instead, thanks.

**Review by AKarmanov — APPROVED** [review]

**Pull request merged**